test/e2e-v2/cases/php/expected/metrics-has-value.yml

```diff
@@ -15,5 +15,11 @@
 
 {{- contains . }}
 - key: {{ notEmpty .key }}
-  value: {{ ge .value 2 }}
+  value:
+    value: 0
+    isemptyvalue: true
```

Comment:
Why do we need to assert that there is empty value?

Reply:
I think the point is, from a test perspective, the initial query should cover the time range from no telenmetry data to having data.